bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder.java

```diff
@@ -1366,6 +1367,18 @@ public boolean getUnselectedImageVisible() {
 	checkWidget();
 	return showUnselectedImage;
 }
+/**
+ * Returns <code>true</code> if an image appears
+ * in selected tabs.
+ *
+ * @return <code>true</code> if an image appears in selected tabs
+ *
+ * @since 3.125
```

Comment:
Shouldn't the Manifest and the Pom also be updated in this case?

Reply:
The required version bump originally was part of this PR: https://github.com/eclipse-platform/eclipse.platform.swt/commit/cf4691d7d546f70b038a0c9b9bf74fe5603c219f#diff-6485b884863fd4f54a20976acef942fe0dd257eb4e72ca25533fd84b7b6fe2e6

But it obviously got lost. Interesting that the API tools baseline comparison did not complain.

I provided #944 to have the faulty version fixed soon.

Reply:
Fixed

Reply:
Here is related snippet from build.
https://ci.eclipse.org/releng/job/eclipse.platform.swt/job/PR-785/12/consoleText
@laeubi : I don't see API tools reporting anything except there is no baseline??

```
[2023-12-14T11:04:05.054Z] [INFO] --- tycho-source:4.0.5-SNAPSHOT:plugin-source (plugin-source) @ org.eclipse.swt ---
[2023-12-14T11:04:05.054Z] [INFO] 
[2023-12-14T11:04:05.054Z] [INFO] --- tycho-packaging:4.0.5-SNAPSHOT:package-plugin (default-package-plugin) @ org.eclipse.swt ---
[2023-12-14T11:04:05.311Z] [INFO] Building jar: /home/jenkins/agent/workspace/eclipse.platform.swt_PR-785/eclipse.platform.swt/bundles/org.eclipse.swt/target/org.eclipse.swt-3.124.300-SNAPSHOT.jar
[2023-12-14T11:04:05.577Z] [INFO] 
[2023-12-14T11:04:05.577Z] [INFO] --- tycho-p2:4.0.5-SNAPSHOT:p2-metadata-default (default-p2-metadata-default) @ org.eclipse.swt ---
[2023-12-14T11:04:05.578Z] [INFO] No baseline version MavenProject: org.eclipse.swt:org.eclipse.swt:3.124.300-SNAPSHOT @ /home/jenkins/agent/workspace/eclipse.platform.swt_PR-785/eclipse.platform.swt/bundles/org.eclipse.swt/pom.xml
[2023-12-14T11:04:05.578Z] [INFO] 
[2023-12-14T11:04:05.578Z] [INFO] --- tycho-p2-extras:4.0.5-SNAPSHOT:compare-version-with-baselines (compare-attached-artifacts-with-release) @ org.eclipse.swt ---
[2023-12-14T11:04:05.578Z] [INFO] Adding repository https://download.eclipse.org/eclipse/updates/4.30/R-4.30-202312010110
[2023-12-14T11:04:05.835Z] [INFO] 
[2023-12-14T11:04:05.835Z] [INFO] --------< org.eclipse.swt:org.eclipse.swt.fragments.localbuild >--------
```

Reply:
>  don't see API tools reporting anything except there is no baseline??

SWT projects have no comparable baseline because of the way they are build. This might change when LSF is enabled (or submodules) and we have one repository as then it would be possible to transform them into regular (tycho) projects.

Reply:
Some work and expermients needs to be done, but I have the intention to do that during the Christmas new-years holidays.

Reply:
Great to hear that you will work on that, @HannesWell, but don't forget to also use holidays as holidays 🙂